pkg/traversal/traversal.go

```diff
@@ -71,7 +45,32 @@ func (w *FileWalker) Traverse() {
 		errorWg.Done()
 	}(errorCh)
 
-	w.processDir(w.reader.Root(), "", queue, errorCh)
+	err := filepath.WalkDir(w.inputPath, func(path string, dirEntry fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if !dirEntry.IsDir() {
+			// the rest of the logic expects the path to be relative to the input dir root, if it fails we assume it is already relative
+			relPath, err := filepath.Rel(w.inputPath, path)
+			if err != nil {
+				queue <- WorkerInput{
```

Comment:
When there is an error shouldn't this be written into the error channel?

Reply:
When and how would it fail?

Reply:
I had that failing yesterday when I ran this end2end, I'll add a test to verify for regressions

Reply:
added